### DISCUSSION TITLE
An proposal for add `series` into other language in i18n

### DISCUSSION THREAD

**Discussion by gitaeron** [Lednerb/bilberry-hugo-theme#470] [Ideas]
According to the 

- discussion-#450
- issue-#451
- commit - #452 at HEAD 355d2b289143cf444560ad7ccb29b9742130fdf5

As My opinion is that we could add series into other languages so that every user could able to use `series` this feature.

Or at least mentioned this in somewhere more specifictly and tell them is an option that could added by themself. 

Perhaps not every user has the know-how about it ? I don't know, but there's might no further question as long as the specifict languages could transfer to ASCII character. 

Although it need a test.

**Comment by gitaeron** [comment]
If I'm wrong or any misunderstanding here, you're welcome to correct me. ヾ(^▽^*)

**Reply by igor-baiborodine** [reply]
@gitaeron, you can submit a pull request if you see missing terms in an i18n file of your choice.
